packages/@dcl/inspector/src/lib/data-layer/host/fs-utils.ts

```diff
@@ -41,5 +41,5 @@ export function getFileName(fileName: string, ext: string) {
 
 export function getCurrentCompositePath(sceneProvider: SceneProvider) {
   const scenePath = sceneProvider.getScene().display.title
-  return withAssetDir(`${scenePath}/main.composite`)
+  return withAssetDir(`${scenePath}/main.composite`).toLowerCase()
```

Comment:
don't we have a normalizePath function over there? the effect is the same but the semantic meaning would add a lot to the devEx

Reply:
or maybe `withAssetDir` could take care of the normalization

Reply:
changed everything to use methods from `path` module instead. We want to allow paths with uppercase letters, but `.composite` files are mandatory to be lowercase